zipline/pipeline/data/dividends.py

```diff
@@ -0,0 +1,25 @@
+"""
+Dataset representing dates of upcoming dividends.
+"""
+from zipline.utils.numpy_utils import datetime64ns_dtype, float64_dtype
+
+from .dataset import Column, DataSet
+
+
+class DividendsByExDate(DataSet):
+    next_ex_date = Column(datetime64ns_dtype)
```

Comment:
should this just be `(next|previous)_date`, we know it is ex_date from the dataset